packages/grid/_modules_/grid/utils/paramsUtils.ts

```diff
@@ -60,8 +62,14 @@ export function buildGridCellParams({
     row: rowModel,
     colDef,
     rowIndex,
+    colIndex: colIndex || (colDef && api.getColumnIndex(colDef.field, true)),
     api,
   };
+  const isEditableAttr = element && element.getAttribute('data-editable');
+  params.isEditable =
```

Comment:
I'm not sure that using the DOM to store a cache is what's ideal. Maybe we could have stored this in the row model.

Reply:
the issue also of using rowModel is that it tightly bind the states to the features.
So in the future, we would want to recompose the grid with hooks...So if you use the rowModels you predefine the state structure with some states that might not be used. It will also be difficult to extend without modifying the model.
ie if you don't use `useEditRows` then you'll have all states in the row models whereas if you use a lookup in the the state then, if you don't use the feature then the state is empty... And with this approach, we can easily creates more hook and states and compose the grid as we want.

Reply:
I was also looking at it. Seems it is done for other things. I'm also not sure if we need it since it is stored in the state.

Reply:
We could have the rowModel a private object with an `any` state. Wouldn't this solve the problem?